KLayout package is broken

See https://github.com/hdl/conda-eda/actions/runs/7994070294/job/21831288262
```
  12:18:52 | ../../../src/tl/tl/tlGit.cc:32:10: fatal error: git2.h: No such file or directory
  12:18:52 |    32 | #include <git2.h>
```
/cc @joamatab

--- a/misc/klayout/meta.yaml
+++ b/misc/klayout/meta.yaml
@@ -45,6 +45,7 @@ requirements:
     - ruby
     - qt >=5.0,<6.0
     - zlib
+    - libgit2
   run:
     - {{ pin_compatible('python', min_pin='x.x', max_pin='x.x') }}
     - ruby

